server/src/test/java/org/elasticsearch/immutablestate/ImmutableClusterStateHandlerTests.java

```diff
@@ -44,53 +42,6 @@ public TransformState transform(Object source, TransformState prevState) throws
         );
     }
 
-    public void testAsMapAndFromMap() throws IOException {
```

Comment:
Not needed anymore. After I refactored the parsing and typed the handlers by the type they receive in transform, I don't need the mapTo methods anymore